src/Stryker.Core/Stryker.Core/ProjectComponents/ProjectComponent.cs

```diff
@@ -60,6 +60,12 @@ public double GetMutationScore()
             double totalCount = TotalMutants.Count();
             double killedCount = DetectedMutants.Count();
 
+            if (totalCount == 0)
+            {
+                // prevent devide by zero
```

Comment:
We should not prevent a divide by zero as NaN **is** a valid value for the mutation score if there are no mutants. The problem is that we are counting the detected wrong.